bindings/cpp/AndroidSecureStorage/app/src/main/cpp/native-lib.cpp

```diff
@@ -0,0 +1,95 @@
+#include <jni.h>
```

Comment:
this seems to copy what's in bindings/cpp/src/android.cc ?

Reply:
this android project is just for generationg *.class file for secure storage api.
as a jar file, it's added to the final project.

Reply:
yes, but for just generating a jar file/class files, there's no need for the entire project with empty unit tests etc. -- just the source files + gradle build manifest with dependencies should be fine?

Reply:
ok, i'll tidy up and keep only important files